pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -32,12 +32,14 @@ const (
 	FullPCPUsOnlyOption            string = "full-pcpus-only"
 	DistributeCPUsAcrossNUMAOption string = "distribute-cpus-across-numa"
 	AlignBySocketOption            string = "align-by-socket"
+	SpreadPCPUsPreferredOption     string = "spread-pcpus-preferred"
```

Comment:
In the KEP we agreed to call this option `distribute-cpus-across-cores` to remain symmetrical with `distribute-cpus-across-numa`:
https://github.com/kubernetes/enhancements/pull/4177/files#diff-a8a9560458bb98bb3b6b12a1f0c81268e03724b9e762124588c3215d5c87f21aR177

Reply:
I will make the change.